android/src/main/java/com/oney/WebRTCModule/WebRTCModule.java

```diff
@@ -522,6 +522,19 @@ public void mediaStreamTrackSetEnabled(final String id, final boolean enabled) {
     }
 
     @ReactMethod
+    public void mediaStreamTrackSwitchCamera(final String id) {
+        MediaStreamTrack track = mMediaStreamTracks.get(id);
+        if (track != null && track.kind().equals("video")) {
```

Comment:
This one applies to the implementations of `mediaStreamTrackSwitchCamera` on both Android and iOS. The check whether track.kind is video has already been taken care of on the JS side and doesn't need to be repeated in the native side. If there's a problem in the source code which causes the check to pass through the JS side but then fail at the native side, I think it's better for us as developers to have it fail on the native side more glamorously than silently because the problem will be in the implementation. If we just silently allow the method call to do nothing, I think it will be a tad difficult to capture the problem at development time. So I'd personally remove the `kind` check on the native side.